proto/process/agent.proto

```diff
@@ -49,6 +49,22 @@ message CollectorProc {
 	ContainerHostType containerHostType = 12;
 }
 
+message CollectorProcDiscovery {
+	string hostName = 1;
+	int32  groupId = 2;
+	int32 groupSize = 3;
+	repeated ProcessDiscovery processDiscoveries = 4;
+	Host host = 5;
+}
+
+message ProcessDiscovery {
```

Comment:
I actually think the file separates `Message Types` (i.e. `Collector*`) from `Models` (e.g. `Process`, `Command`, etc). If you ctrl-F for `// Models` you'll see the comment that begins the Models section. It does look like `CollectorStatus` is probably in the wrong place, heh